cfgov/unprocessed/apps/youth-employment-success/js/budget-form-view.js

```diff
@@ -0,0 +1,91 @@
+import { checkDom, setInitFlag } from '../../../js/modules/util/atomic-helpers';
+import inputView from './input-view';
+import money from './money';
+
+const CLASSES = Object.freeze( {
+  FORM: 'o-yes-budget',
+  EARNED_INPUT: 'o-yes-budget-earned',
+  SPENT_INPUT: 'o-yes-budget-spent',
+  REMAINING: 'o-yes-budget-remaining'
+} );
+
+/**
+ * BudgetFormView
+ * @class
+ *
+ * @classdesc Initializes the organism.
+ *
+ * @param {htmlnode} element
```

Comment:
TRIVIAL: this should be [HTMLElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement). We have this wrong elsewhere. The parent interface of HTMLElement is Node.

Reply:
Will hcange it back